### ISSUE TITLE
Build failure when doing `make docs` for Julia in Windows

### ISSUE THREAD

**Issue by peteristhegreat** [JuliaTime/TimeZones.jl#253]
In the middle of `make binary-dist` for Julia v1.3.1, it blows up because of TimeZones not able to find `7z.exe` (its in the `make docs` part).

```
Building HTML documentation.
  Updating registry at `C:\development\julia_v1.3.1\doc\deps\registries\General`
  Updating git-repo `https://github.com/JuliaRegistries/General.git`
 Installed IniFile ───────────── v0.5.0
 Installed HeaderREPLs ───────── v0.3.0
 Installed ArgParse ──────────── v0.6.2
 Installed BufferedStreams ───── v1.0.0
 Installed JuliaInterpreter ──── v0.7.9
 Installed TextWrap ──────────── v0.3.0
 Installed Documenter ────────── v0.24.2
 Installed Memento ───────────── v0.12.1
 Installed CodeTracking ──────── v0.5.8
 Installed XML2_jll ──────────── v2.9.9+1
 Installed LibCURL ───────────── v0.5.2
 Installed JSON ──────────────── v0.21.0
 Installed Revise ────────────── v2.2.2
 Installed Libiconv_jll ──────── v1.16.0+1
 Installed Compat ────────────── v2.2.0
 Installed Rebugger ──────────── v0.3.2
 Installed Filetimes ─────────── v0.1.0
 Installed OrderedCollections ── v1.1.0
 Installed Mocking ───────────── v0.7.1
 Installed BinDeps ───────────── v1.0.0
 Installed LibExpat ──────────── v0.6.1
 Installed Libz ──────────────── v1.0.1
 Installed DocStringExtensions ─ v0.8.1
 Installed Syslogs ───────────── v0.3.0
 Installed Expat_jll ─────────── v2.2.7+0
 Installed URIParser ─────────── v0.4.0
 Installed PackageCompiler ───── v0.6.3
 Installed HTTP ──────────────── v0.8.6
 Installed WinRPM ────────────── v0.4.3
 Installed DataStructures ────── v0.17.2
 Installed Zlib_jll ──────────── v1.2.11+8
 Installed BinaryProvider ────── v0.5.8
 Installed EzXML ─────────────── v1.1.0
 Installed HTTPClient ────────── v0.2.1
 Installed Parsers ───────────── v0.3.11
 Installed ExprTools ─────────── v0.1.0
 Installed LoweredCodeUtils ──── v0.3.8
 Installed TimeZones ─────────── v1.0.0
  Building LibCURL ────────→ `C:\development\julia_v1.3.1\doc\deps\packages\LibCURL\lWJxD\d
eps\build.log`
  Building WinRPM ─────────→ `C:\development\julia_v1.3.1\doc\deps\packages\WinRPM\BfpQI\de
ps\build.log`
  Building PackageCompiler → `C:\development\julia_v1.3.1\doc\deps\packages\PackageCompiler
\oT98U\deps\build.log`
  Building TimeZones ──────→ `C:\development\julia_v1.3.1\doc\deps\packages\TimeZones\vYAiM
\deps\build.log`
┌ Error: Error building `TimeZones`:
│ [ Info: Downloading 2019c tzdata
│ ERROR: LoadError: IOError: could not spawn `'C:\development\julia_v1.3.1\usr\bin\..\libex
ec\7z.exe' t 'C:\development\julia_v1.3.1\doc\deps\packages\TimeZones\vYAiM\deps\tzarchive\
tzdata2019c.tar.gz' -y`: no such file or directory (ENOENT)
│ Stacktrace:
│  [1] _spawn_primitive(::String, ::Cmd, ::Array{Any,1}) at .\process.jl:99
│  [2] #554 at .\process.jl:112 [inlined]
│  [3] setup_stdios(::Base.var"#554#555"{Cmd}, ::Array{Any,1}) at .\process.jl:196
│  [4] success(::Cmd) at .\process.jl:111
│  [5] isarchive at C:\development\julia_v1.3.1\doc\deps\packages\TimeZones\vYAiM\src\tzdat
a\archive.jl:29 [inlined]
│  [6] tzdata_download(::String, ::String) at C:\development\julia_v1.3.1\doc\deps\packages
\TimeZones\vYAiM\src\tzdata\download.jl:93
│  [7] #build#32(::Bool, ::typeof(TimeZones.TZData.build), ::String, ::Array{String,1}, ::S
tring, ::String, ::String) at C:\development\julia_v1.3.1\doc\deps\packages\TimeZones\vYAiM
\src\tzdata\build.jl:46
│  [8] #build at .\none:0 [inlined]
│  [9] build(::String) at C:\development\julia_v1.3.1\doc\deps\packages\TimeZones\vYAiM\src
\tzdata\build.jl:82
│  [10] #build#14(::Bool, ::typeof(build), ::String) at C:\development\julia_v1.3.1\doc\dep
s\packages\TimeZones\vYAiM\src\build.jl:11
│  [11] build at C:\development\julia_v1.3.1\doc\deps\packages\TimeZones\vYAiM\src\build.jl
:11 [inlined] (repeats 2 times)
│  [12] top-level scope at C:\development\julia_v1.3.1\doc\deps\packages\TimeZones\vYAiM\de
ps\build.jl:3
│  [13] include at .\boot.jl:328 [inlined]
│  [14] include_relative(::Module, ::String) at .\loading.jl:1105
│  [15] include(::Module, ::String) at .\Base.jl:31
│  [16] include(::String) at .\client.jl:424
│  [17] top-level scope at none:5
│ in expression starting at C:\development\julia_v1.3.1\doc\deps\packages\TimeZones\vYAiM\d
eps\build.jl:3
└ @ Pkg.Operations C:\development\julia_v1.3.1\usr\share\julia\stdlib\v1.3\Pkg\src\backward
s_compatible_isolation.jl:649
ERROR: LoadError: ArgumentError: Package Documenter [e30172f5-a6a5-5a46-863b-614d45cd2de4]
is required but does not seem to be installed:
 - Run `Pkg.instantiate()` to install all recorded dependencies.

Stacktrace:
 [1] _require(::Base.PkgId) at .\loading.jl:993
 [2] require(::Base.PkgId) at .\loading.jl:922
 [3] require(::Module, ::Symbol) at .\loading.jl:917
 [4] include at .\boot.jl:328 [inlined]
 [5] include_relative(::Module, ::String) at .\loading.jl:1105
 [6] include(::Module, ::String) at .\Base.jl:31
 [7] exec_options(::Base.JLOptions) at .\client.jl:287
 [8] _start() at .\client.jl:460
in expression starting at C:\development\julia_v1.3.1\doc\make.jl:9
make[3]: *** [Makefile:42: html] Error 1
make[3]: Leaving directory '/cygdrive/c/development/julia_v1.3.1/doc'
make[2]: *** [Makefile:91: docs] Error 2
make[2]: Leaving directory '/cygdrive/c/development/julia_v1.3.1'
make[1]: *** [/cygdrive/c/development/julia_v1.3.1/Makefile:47: /cygdrive/c/development/julia_v1.3.1/doc/_build/html/en/index.html] Error 2
make[1]: Leaving directory '/cygdrive/c/development/julia_v1.3.1'
make: *** [Makefile:451: binary-dist] Error 2
```

I fixed it by copying `7z.exe` and `7z.dll` from (in the julia checkout folder) `./usr/bin` to `./usr/libexec`, then it would work.

It is odd that `TimeZones` is getting brought in (I couldn't find it in the `Manifest.toml`).  I think `Base.download` is getting better, but in case someone needs it, this is a fix that works.

**Comment by omus** [comment]
Related: https://github.com/JuliaTime/TimeZones.jl/issues/218

**Comment by omus** [comment]
I think we can fix this problem in TimeZones by checking both `./usr/bin` and `./usr/libexec` but an update would also have to happen in Julia to use a newer release. I'll note version 1.0.1 has been out for 2 months and this used 1.0.0.

**Comment by omus** [comment]
I asked on the Julia Slack and this is a known bug on Julia 1.3.1 which is fixed in Julia 1.4.0 and should be backported to 1.3.2 if that happens.

Reference: https://github.com/JuliaLang/julia/commit/e0f904504f4fcd9388ec5da50380d9a28d9086a3

**Comment by omus** [comment]
I'll close this as there's nothing left to be done.
